app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -253,7 +253,7 @@ def quota_find_prov_request_by_group(options)
 
   def quota_find_active_prov_request_by_owner(options)
     email = get_option(:owner_email).to_s.strip
-    quota_find_active_prov_request(options).select { |p| email.casecmp(p.request_owner_email(p)).zero? }
+    quota_find_active_prov_request(options).select { |p| email.casecmp(p.request_owner_email(p)).zero? } if email.present?
```

Comment:
When is this condition true?  Shouldn't we always have an email?

Reply:
QE tested this and when tenant is user, the owner email was nil.  If you login as admin and haven't configured an email for the admin user then owner email could be nil.  This was only failing with tenant set to user.